composeApp/src/androidMain/kotlin/it/fast4x/rimusic/ui/components/themed/PlaylistsItemGridMenu.kt

```diff
@@ -503,6 +505,31 @@ fun PlaylistsItemGridMenu(
                     )
                 }
 
+                onEditThumbnail?.let { onEditThumbnail ->
```

Comment:
These two are identical, except for the `title`. Having two similar blocks of code makes it harder to read and harder to maintain.

I can suggest a solution
```kotlin
var title: Int? = null
if( onEditThumbnail != null )
    title = R.string.edit_thumbnail
if( onResetThumbnail != null )
    title = R.string.reset_thumbnail

title ?: return // Or something that skip next line

GridMenuItem(
    icon = R.drawable.image,
    title = title,
    colorIcon = colorPalette.text,
    colorText = colorPalette.text,
    onClick = {
        onDismiss()
        onEditThumbnail()
    }
)
```

Reply:
1. This code snippet doesn't allow both options simultaneously, right?
2. They differ in the callback function too, not just the title.
3. It's kind of repetitive, I agree, but thats how all the other options were written